### ISSUE TITLE
Problems with Claas Saddle trac

### ISSUE THREAD

**Issue by wisco01** [stijnwop/manualAttach#118]
Although I can contact the CLAAS SADDLE TRAC, but I can not connect the hoses and then do not drive with that and then you can only switch on and off only if you have admin rights and I do not want to give each admin pasception on the server

Translated by google

**Comment by Pleasure0102** [comment]
can confirm, no way to connect hoses it seems, vehicle is moving slowly, not working correctly. I tried without MA and it does not seem it actually connects the hoses, but works more faster, to reproduce, try to connect the saddle trac with Xerion saddle trac tank

**Comment by wisco01** [comment]
I have now tested it with the small one, with the small one there are no problems, only with the large tank

**Comment by ElLoko** [comment]
was also running in this issue. you can attach the trailer but not the hoses.

**Comment by TheKelric** [comment]
I can confirm the problem. wanted to attach the trailer that works. it comes the option that you can press y for the hoses. but they do not connect.

**Comment by idespaiu** [comment]
I noticed the same issue with the Ploeger slurry tanker

**Comment by stijnwop** [comment]
Yes thats cause the Ploeger isn't setup correctly.. (it's missing an input for electric)

Somewhere in the future I will try to detect specific hoses not being attached and not disable everything when 1 is not able to be attached.. which is the case for the Ploeger.

**Comment by qqqk2** [comment]
There is a problem if the KAWECO FRONT-UNIT is not connected. If you connect both the barrel and the FRONT-UNIT, and then manually connect the hoses, it works. But start connecting the hoses only on the barrel - this is how everything works, except for the FRONT-UNIT (it does not turn around and there is no light). in any case - there is no animation of connecting hydraulics, electricity and air. It seems that the hose textures are not registered, although on the XERION 4200 SADDLE TRAC dots (fittings) are visible in front and behind.

**Comment by KITT3000** [comment]
The problem has been solved in the latest version.
